language-plutus-core/src/Language/PlutusCore/Pretty/ConfigName.hs

```diff
@@ -0,0 +1,117 @@
+module Language.PlutusCore.Pretty.ConfigName
+    ( PrettyConfigName (..)
+    , HasPrettyConfigName (..)
+    , defPrettyConfigName
+    , debugPrettyConfigName
+    ) where
+
+{- Note [Update on PLC names]
+We no longer have such fancy names that Note [PLC names pretty-printing] describes.
+However it's still nice to have a working boileplate-free solution for sophisticated cases.
+-}
```

Comment:
Note the note.

Reply:
I think this is better as a paragraph on top of the note it references: someone reading that will not necessarily see this note and may be confused.

Reply:
I also updated the original note to refer to the new one, but yeah, probably it's just better to keep a single note.